nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2157,7 +2325,10 @@ async fn routers_routes_post(
 }
 
 /**
- * Delete a route from its router
+ * Delete a route
+ *
+ * Permanently deletes the specified route from the specified router.
+ * TODO: We might want some more detail here on the behavior of this endpoint.
```

Comment:
Yea, related to my other comment we can only add/delete routes from VPC Custom Routers.

Reply:
will elaborate